huaweicloud/services/acceptance/sfs/resource_huaweicloud_sfs_turbo_dir_test.go

```diff
@@ -0,0 +1,90 @@
+package sfs
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func sfsTurboDirReadfunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	var (
+		region  = acceptance.HW_REGION_NAME
+		httpUrl = "v1/{project_id}/sfs-turbo/shares/{share_id}/fs/dir"
+	)
+	sfsClient, err := cfg.SfsV1Client(region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating SFS v1 client: %s", err)
+	}
+
+	getPath := sfsClient.Endpoint + httpUrl
+	getPath = strings.ReplaceAll(getPath, "{project_id}", sfsClient.ProjectID)
+	getPath = strings.ReplaceAll(getPath, "{share_id}", state.Primary.Attributes["share_id"])
+	getPath += fmt.Sprintf("?path=%s", state.Primary.ID)
+	getOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	getResp, err := sfsClient.Request("GET", getPath, &getOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving SFS file directory %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error retrieving SFS Turbo file directory %s", err)
```
Please check the other logging parts.